styles.css

```diff
@@ -77,28 +96,404 @@ p {
-  font-weight: 600;
-  color: #6070ff;
-}
-
-.hero-text p {
+.banner .banner-text p {
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   color: #344563;
+  margin-top: 10px;
+  margin-bottom: 12px;
+}
+
+/* Connect style */
+.link {
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
 }
 
+/* Social Media styles */
 .social {
   display: flex;
+  margin-top: 12px;
   gap: 20px;
+}
+
+.social-nl a {
+  margin-right: 10px;
+  color: #505f79;
+  font-size: 25px;
+}
+
+.social a:hover {
+  color: #505f79;
+  opacity: 0.5;
+}
+
+/* Work  */
+.work {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-top: 50px;
+}
+
+.social-nl a:hover {
+  color: #505f79;
+  opacity: 0.5;
+}
+
+.work .lang a {
+  padding: 5px 10px;
+  margin-right: 10px;
+  background: #ebebff;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  border-radius: 8px;
+}
+
+.work .card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 20px;
+  margin-top: 104px;
+  border: 1px solid #e6e8eb;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.1);
+}
+
+.lan {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
+}
+
+.lan h2 {
+  font-weight: 500;
+  font-size: 20px;
+}
+
+.skill-lang li h2 {
+  font-weight: 500;
+  font-size: 15px;
+  color: #253858;
+  margin-top: 8px;
+}
+
+.work .card-right h2 {
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.work .card .cano {
+  display: flex;
+  margin-top: 10px;
+}
+
+.skill-lang li {
+  display: flex;
+  justify-content: start;
+  background: #f7f7f9;
+  border-radius: 8px;
+  width: 100%;
+  gap: 10px;
+  margin: 5px 0;
+  padding: 10px;
+}
+
+.work .lang li {
+  margin-bottom: 10px;
+}
+
+.work .card .cano li {
+  margin-right: 10px;
+  font-weight: 600;
+  font-size: 13px;
+  color: #7a869a;
+}
+
+.skills .skill li span i {
+  font-size: 15px;
+  color: #505f79;
+}
+
+.work .card .cano li i {
+  color: #7a869a;
+  font-size: 10px;
+  margin-left: 10px;
+}
+
+.work .card .cano li:first-child {
+  color: #344563;
+}
+
+.work .card .cano li:first-child i {
+  color: #7a869a;
+  font-size: 10px;
+  margin-left: 10px;
+}
+
+.work .card p {
+  font-size: 15px;
+  color: #344563;
+}
+
+.work .lang {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.work .card-left-desk {
+  display: none;
+}
+
+.btn-default {
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 126px;
+  height: 48px;
+  background: #fff;
+  font-weight: 500;
+  font-size: 17px;
   margin-top: 10px;
+  color: #396df2;
+}
+
+.btn-white {
+  font-family: "Poppins", sans-serif;
+  border: 1px solid #fff;
+  border-radius: 8px;
+  width: 126px;
+  height: 48px;
+  background: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  margin-top: 10px;
+  color: #396df2;
+}
+
+.btn-default:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.btn-default:hover {
+  background-color: #6070ff;
+  color: #b3b7d4;
+}
+
```

Comment:
- [ ] Good job for adding the button interactions 🥇 However, the text color for hover interaction is not according to the given design guidelines. Kindly use `#fff` for text color. Also I noticed that you missed the `box-shadow` property for hover interaction. Kindly fix this as mentioned by the previous reviewer [here](https://github.com/Srilearns/Portfolio-Website-Mobile-First/pull/3#discussion_r1256911920) ⛏🤓 

Your hover interaction | Given design 
---| ---|
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/106119105/b021fb84-f904-4c5e-9a56-5b00cc56d268) | ![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/106119105/e7039036-9a73-45bf-9209-4ee60b12b19a)

_Please make sure to apply this for all buttons_

Reply:
Fixed.